CMakeLists.txt

```diff
@@ -6,7 +6,7 @@
 cmake_minimum_required(VERSION 3.23)
 project(
   BSMPT
-  VERSION 3.0.5
+  VERSION 3.0.6
```

Comment:
```suggestion
  VERSION 3.0.5
```

Reply:
I don't think we need to increase the version for this one. There is no change in the actual code but only a utility script